examples/reactnative/rn063example/README.md

```diff
@@ -1,9 +1,13 @@
 # React Native
 
+For instructions on how to install and configure Bugsnag in your own application please consult our React Native [documentation](https://docs.bugsnag.com/platforms/react-native/react-native/). 
+
 This is an example project showing how to use `@bugsnag/react-native` with a React Native v0.63 project.
 
 This project was bootstrapped with `react-native init`.
```

Comment:
I'm not sure this section reads very well in this order - can you move the first sentence to the end so that we start by saying what the example is?

```suggestion
This is an example project showing how to use `@bugsnag/react-native` with a React Native v0.63 project.

This project was bootstrapped with `react-native init`.

For instructions on how to install and configure Bugsnag in your own application please consult our React Native [documentation](https://docs.bugsnag.com/platforms/react-native/react-native/). 
```